cluster/src/main/scala/io/snappydata/ToolsCallbackImpl.scala

```diff
@@ -117,6 +117,7 @@ object ToolsCallbackImpl extends ToolsCallback with Logging {
         case ite: InvocationTargetException => assert(ite.getCause.isInstanceOf[SecurityException])
       }
     }
+    Thread.currentThread().setContextClassLoader(lead.urlclassloader)
```

Comment:
I don't think this is the right fix. Please test with other jars and packages which is passed through the job. With those a classloader is created which is specific to those coordinates and jars passed. They will be ignored.